mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager.go.erb

```diff
@@ -216,6 +216,49 @@ func ResourceComputeInstanceGroupManager() *schema.Resource {
 				},
 			},
 
+			<% unless version == "ga" -%>
+			"standby_policy": {
+				Computed:    true,
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: `Standby policy for stopped and suspended instances.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"initial_delay_sec": {
+							Type:             schema.TypeInt,
+							Default:          0,
+							Optional:         true,
+							ValidateFunc:     validation.IntBetween(0, 3600),
+							Description:      `Specifies the number of seconds that the MIG should wait to suspend or stop a VM after that VM was created. The initial delay gives the initialization script the time to prepare your VM for a quick scale out. The value of initial delay must be between 0 and 3600 seconds. The default value is 0.`,
+						},
+
+						"mode": {
+							Type:         schema.TypeString,
+							Default:      "MANUAL",
+							Optional:     true,
+							ValidateFunc: validation.StringInSlice([]string{"MANUAL", "SCALE_OUT_POOL"}, true),
+							Description:  `Defines how a MIG resumes or starts VMs from a standby pool when the group scales out. The default mode is "MANUAL".`,
+						},
+					},
+				},
+			},
+
+			"target_suspended_size": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Optional:    true,
+				Description: `The target number of suspended instances for this managed instance group.`,
+			},
+
+			"target_stopped_size": {
```

Comment:
Is there a default value returned in the API response if no value is sent in the API request?

Reply:
If we don't set this value to api request, it will return the calculated value which was already set before the api request was called.